app/jobs/regular/enable_bootstrap_mode.rb

```diff
@@ -0,0 +1,24 @@
+module Jobs
+  class EnableBootstrapMode < Jobs::Base
+    sidekiq_options queue: 'critical'
+
+    def execute(args)
+      raise Discourse::InvalidParameters.new(:user_id) unless args[:user_id].present?
+      return if SiteSetting.bootstrap_mode_enabled
+
+      @current_user = User.find_by(id: args[:user_id])
```

Comment:
I don't think `@current_user` should be an instance variable. Also it's a little confusing that it uses the same name as the function `current_user` elsewhere.  I'd suggest just calling it `user` instead.